chromium_src/components/content_settings/core/browser/content_settings_registry.cc

```diff
@@ -64,6 +64,38 @@
       ContentSettingsInfo::INHERIT_IN_INCOGNITO, \
       ContentSettingsInfo::PERSISTENT, \
       ContentSettingsInfo::EXCEPTIONS_ON_SECURE_AND_INSECURE_ORIGINS); \
+  /* We need 62, 63, 64 for ContentSettingsTypes for Android tabs migration*/ \
```

Comment:
in brave-core we handle brave-specific content settings by using a content type of plugin and specify the resource id. Even if we did add content setting types, we can't just start at 62 and hope that's will be safe forever

Reply:
or is this strictly for reading the old content settings and they will be written to plugins?